.claude/commands/reviewer-decide.md

```
@@ -9,6 +9,24 @@ Based on steps 1-3, make a decision.
+
+If every changed file is documentation-only (`docs/**` or doc-text files such as `.md`, `.mdx`, `.txt`, `.rst`, `.adoc`), do the standards pass, push any doc fixes, and route straight to `/pr-ready`.
+
+```bash
+gh pr checkout <number>
+# ... improve wording / links / receipts as needed ...
+git push
+gh pr comment <number> --body "Standards review complete. Docs-only fast-track used; no reviewer-deep pass required."
+```
+
+Then call:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)